### ISSUE TITLE
A11y: An empty image presents at the beginning of the component

### ISSUE THREAD

**Issue by neet** [xyflow/xyflow#5190]
### Please describe the feature that you want to propose

This is another tiny problem we have discovered. When you use React Flow with screen readers, it always announces "image" with no extra description at the beginning of the component:

https://github.com/user-attachments/assets/ceb4fb4a-e186-4713-8d7d-ef4928ae8790

I believe this violates [1.1.1 Non-text Content from WCAG 2.1](https://www.w3.org/TR/WCAG21/#non-text-content). Although is not critical, it still spoils user experience for screen readers. 

I identified that this is caused by a `<svg>` element with class `react-flow__marker`. I verified that when I put `aria-hidden="true"` to the `react-flow__marker`, the empty "image" announcement gets skipped.

I recorded the video with VoiceOver in macOS 15.3.2 and the latest Chrome.

**Comment by moklick** [comment]
Thanks! That's an easy thing to fix. We will do it asap.

**Comment by moklick** [comment]
fixed in 12.5.5
